jphoenix86/real_time
# 7 add sockets server connects_to #7

@rrgayhart, @stevekinney, @applegrain
#### What's this PR do?

This PR gives a user the ability to be able to go through create a survey (obviously its a little bit 1.0 ish).
When a user now clicks create survey it should create a nested object which is the survey including the options they want. This should then redirect them directly to their admin panel where the buttons are the things they just entered.
#### Where should the reviewer start?

Navigate to: '/'
Give your survey a name.
Click the link to add fields until your survey is the desired length.
Click create survey.
#### How should this be manually tested?

Should be as easy as:

npm install
npm start
navigate to root and try things out!
#### Any background context you want to provide?

As a note to the above I have not yet implemented a DB or any way to maintain state over time. So if you leave a page and refresh it in the wrong way then you will lose the thing.
#### What gif best describes this PR or how it makes you feel?

![batman](http://i.giphy.com/26CalmkpfFU3io3yU.gif)
